apps/design/frontend/src/system_settings_screen.tsx

```diff
@@ -223,6 +223,25 @@ export function SystemSettingsForm({
               }
               disabled={!isEditing}
             />
+            <CheckboxGroup
+              label="VxAdmin"
+              options={adjudicationReasonOptions.filter((option) =>
+                [AdjudicationReason.MarginalMark].includes(option.value)
```

Comment:
For now I've only added `MarginalMark` as a check-able item, discussion [here](https://github.com/votingworks/vxsuite/issues/6371#issuecomment-2880704695)